src/pages/grapple.mdx

```diff
@@ -0,0 +1,60 @@
+import '../styles/_defaults.scss'
+import '../styles/style.scss'
+
+<title>Grappling</title>
+
+# How to Grapple
+
+[Home](/)
+
+Ready to rassle?
+
+### In order to grapple you must:
+
+- Use the attack action to make a special melee attack (the grapple).
+    - If you have multiple attacks, this replaces one of them.
+- Be no more than one size smaller than your target.
+- Have at least one free hand.
+- Make a **Strength(Athletics)** check contested by the targets **Strength(Athletics)** or **Dexterity(Acrobatics)** check.
+
+If you succeed, the target is subject to the [Grappled](/conditions) condition. It is a free action to release the target.
```

Comment:
Should this be a link on the word `condition` instead of `Grapple`?

Reply:
Grappled is the condition that one would follow a link to learn about, so Grappled should be the link. 
I can add an anchors in the conditions page so that we can link directly to the specific condition. Would that make more sense?

Reply:
Oh, right. Imo, both Grapple and condition should be linked, but this makes sense.

Reply:
You think they should _both_ be linked?

Reply:
Can you clarify/elaborate? Are you saying you think "Grappled condition" should be the link? Or that "Grappled" and "condition" should each be linked?

Reply:
I think "Grappled condition" should be one big link with a heading link if possible. ie. `/conditions#grappled`

Reply:
I've made the changes to link to the specific condition 09d1120

I don't think "condition needs to be a part of the link. Unless you think I should add the word condition after the linked word "Prone" as well, the link being the name of the condition seems consistent and correct to me.

Alternatively, I could change the language to state that "If you succeed the target is Grappled" and remove the word condition. This is largely semantics

![image](https://user-images.githubusercontent.com/10962974/147630358-a1059d6a-e915-487d-9226-52de0cfaaa04.png)
![image](https://user-images.githubusercontent.com/10962974/147630362-bf4d7d84-d98b-4b07-a864-6dae0b024e5e.png)

Reply:
Ah, yeah. That works. All good.